curriculum/challenges/english/14-responsive-web-design-22/learn-css-animation-by-building-a-ferris-wheel/step-015.md

```diff
@@ -92,19 +92,19 @@ assert(new __helpers.CSSHelp(document).getStyle('.cabin:nth-of-type(5)')?.top ==
 You should have a `.cabin:nth-of-type(6)` selector.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.cabin:nth-of-type(6)'));
+assert(new __helpers.CSSHelp(document).getStyle('.cabin:nth-of-type(6)') || new __helpers.CSSHelp(document).getStyle('.cabin:last-of-type'));
 ```
 
 Your `.cabin:nth-of-type(6)` selector should have a `right` property set to `17%`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.cabin:nth-of-type(6)')?.right === '17%');
+assert(new __helpers.CSSHelp(document).getStyle('.cabin:nth-of-type(6)')?.right === '17%' || new __helpers.CSSHelp(document).getStyle('.cabin:last-of-type')?.right === '17%');
 ```
 
 Your `.cabin:nth-of-type(6)` selector should have a `top` property set to `7%`.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('.cabin:nth-of-type(6)')?.top === '7%');
+assert(new __helpers.CSSHelp(document).getStyle('.cabin:nth-of-type(6)')?.top === '7%' || new __helpers.CSSHelp(document).getStyle('.cabin:last-of-type')?.top === '7%');
```

Comment:
```suggestion
const top = (s) => new __helpers.CSSHelp(document).getStyle(s)?.top;
assert.equal(top('.cabin:last-of-type') || top('.cabin:nth-of-type(6)'),  '7%'));
```

Reply:
@ShaunSHamilton I made these changes you suggested along for step-015's 6th element. Thank you for the tips!

Reply:
For future reference, you can use the `Commit suggestion` button to accept a suggestion and commit it. Even better, when there are multiple suggestions, you can go to the `Files` tab, and use the `Add suggestion` to batch button to include all suggestions in a single commit.